tachyon/zk/plonk/permutation/permutation_argument_runner_impl.h

```diff
@@ -38,6 +38,9 @@ PermutationArgumentRunner<Poly, Evals>::CommitArgument(
   // will never underflow because of the requirement of at least a degree
   // 3 circuit for the permutation argument.
   CHECK_GE(constraint_system_degree, argument.RequiredDegree());
+  if (argument.columns().empty()) {
+    return {};
+  }
```

Comment:
Please remove the curly braces for consistency with the code below.
```suggestion
  if (argument.columns().empty()) return {};
```